packages/material-ui-lab/src/LoadingButton/loadingButtonClasses.ts

```diff
@@ -0,0 +1,42 @@
+import { generateUtilityClass, generateUtilityClasses } from '@material-ui/unstyled';
+
+export interface LoadingButtonClasses {
```

Comment:
Oops we should extend the `ButtonClasses` here.

Reply:
I extends the classes in `LoadingButton.d.ts` instead, which make more sense because those classes are meant to be passed to `Button`. (LoadingButtonClasses remains the same)

```ts
// LoadingButton.d.ts
  classes?: Partial<ButtonClasses> & {
    /** Styles applied to the root element. */
    root?: string;
    /** Styles applied to the root element if `loading={true}`. */
    loading?: string;
    ...
  }
```

Reply:
also added `.spec.tsx` to test classes for typescript, which is working fine.